src/shmem_internal.h

```diff
@@ -598,7 +598,8 @@ static inline
 int shmem_internal_pe_in_active_set(int global_pe, int PE_start, int PE_stride, int PE_size)
 {
     int n = (global_pe - PE_start) / PE_stride;
```

Comment:
I see two instances of this function being called in SOS. In one instance (call from `team_translate_pe`) this function is called, where PE_stride is the taken from the destination team. Just want to know, if it is guaranteed that any team should not have a PE_stride = 0;

Reply:
Actually, it doesn't look possible for `PE_stride` to be zero here (because in `team_split_strided`, the check already happened and in `team_translate`, the team was already successfully created).  But it doesn't hurt to check with a non-persistent `shmem_internal_assert`.  I will go with that unless you suggest otherwise, @wrrobin.

Reply:
Thanks @davidozog. Yes, this assert is fine.